### PR TITLE
tests: addition of test_examples_app.py

### LINKED ISSUE #94: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:
- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
- Adds test_examples_app.py. (closes #94)

Signed-off-by: Leonardo Rossi leonardo.r@cern.ch

### CODE REVIEW

**Comment by hachreak** [comment]
@jirikuncar :shipit: ?

File: tests/test_examples_app.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    # stop server
+    os.killpg(webapp.pid, signal.SIGTERM)
+    # tear down example app
+    cmd = 'FLASK_APP={0} ./app-teardown.sh'.format(app_name)
+    subprocess.call(cmd, shell=True)
+    # return to the original directory
+    os.chdir(current_dir)
+
+
+@pytest.yield_fixture
```
Comment (jirikuncar) [inline tests/test_examples_app.py]: What about parametrized test:

```python
@pytest.mark.parametrized('service', ['orcid', 'github', 'cern'])
def test_app(service):
    """Test example app for given service."""
    with _create_example_app('{0}_app.py'.format(service)):
        cmd = 'curl http://localhost:5000/{0}'.format(service)
        output = subprocess.check_output(cmd, shell=True).decode('utf-8')
        assert 'Redirect' in output
```

Comment (hachreak) [inline tests/test_examples_app.py]: Do you mean to not use the yield_fixture of pytest?

Comment (jirikuncar) [inline tests/test_examples_app.py]: You can parameterize the fixture itself. I was just trying to give simple example.

**Comment by hachreak** [comment]
@switowski is it mergeable now? :smile:

**Pull request merged**